src/formats/luma.rs

```diff
@@ -0,0 +1,11 @@
+#[repr(C)]
+#[cfg_attr(feature = "serde", derive(serde::Serialize, serde::Deserialize))]
+#[cfg_attr(feature = "defmt-03", derive(defmt::Format))]
+#[derive(Copy, Clone, Debug, Default, Eq, PartialEq, Ord, PartialOrd, Hash)]
+/// A `Luminosity` pixel.
```

Comment:
Nope, luma is luma. Not a physical property.

If you need a synonym, brightness is vague enough.

Reply:
Whoops, my bad I think I wrote that before I realized there was a difference and `Luma` was not just a short form for `Luminosity` :frown:

Reply:
It seems the only difference between `Luma` and `Luminance` is `Luma` is based on gamma-correction and `Luminosity` is based on linear-`RGB`, in this case, since we are an color-space agnostic library, we don't want to specify either variant? Either we go back to `Gray`, or we stay with `Luma` but describe this as a `Brightness` pixel as you suggested?

Reply:
None of the options is ideal. Gray has only poor options for the field name. .gray not matching the naming scheme, .g looking like green, .l not matching the type name, .0 terrible for alpha-first GA. 
Brightness would have .b clashing with blue, and a long name.

* Light? Probably annoying to game devs where lights are a thing 

* Mono? Technically monochromatic works, although I haven't seen that name used.

Reply:
I've replaced `Luminosity` with `Brightness` and will have to hope that library users will remember that all the library types are gamma correction agnostic.

Reply:
Monochromatic could conflict with 0 or 1 binary images https://en.wikipedia.org/wiki/Binary_image

Oh the struggle with naming hehe

Reply:
`Gray.v` could contradict with a potential `Yuv` pixel type, I'm not sure how big an issue that is though.

That said, I do think `Gray` with some letter would be the best solution, I like `Gray`.v a lot.

Reply:
That or we use `Gray.l` and call it "lightness" in the documentation?